.github/workflows/cpp.yaml

```diff
@@ -0,0 +1,73 @@
+name: C++
+
+on:
+  push:
+    branches: [ v3 ]
+
+jobs:
+  ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        build:
+          - os: ubuntu-16.04
+            cc: gcc
+            cxx: g++
+          - os: ubuntu-16.04
+            cc: clang
+            cxx: clang++
+          - os: ubuntu-18.04
+            cc: gcc
+            cxx: g++
+          - os: ubuntu-18.04
+            cc: clang
+            cxx: clang++
+          - os: ubuntu-20.04
+            cc: gcc
+            cxx: g++
+          - os: ubuntu-20.04
+            cc: clang
+            cxx: clang++
+          - os: macos-10.15
+            cc: gcc
+            cxx: g++
+          - os: macos-10.15
+            cc: clang
+            cxx: clang++
+          # macOS 11 is in private preview currently
+          #- os: macos-11.0
+          # Windows needs to be tuned before it can run, I didn't bother, feel free to contribute
+          #- os: windows-2019
+        # Just 1 Node.js version should be fine for C++
```

Comment:
Actually in package.json we say that mediasoup is supposed to run in `"node": ">=10"`. Can we also test node 10 and 12 here?

Reply:
This is just for worker builds, which do not depend on Node.js really. Node.js workflow file is being tested on Node.js 10, 12, 14 and 15

Reply:
absolutely, sorry